recipes/pthread-win32/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "2.9.1":
+    url: "https://codeload.github.com/GerHobbelt/pthread-win32/zip/19fd5054b29af1b4e3b3278bfffbb6274c6c89f5"
```

Comment:
I did not work on the original bincrafters recipe, so these are guesses:
https://github.com/GerHobbelt/pthread-win32/archive/v-2-9-1-release.zip redirects to https://codeload.github.com/GerHobbelt/pthread-win32/zip/v-2-9-1-release, so I guess this url was used to save an indirection.
the commit `19fd5054b29af1b4e3b3278bfffbb6274c6c89f5`  (October 2015) is used instead of `v-2-9-1-release` tag (May 2012), because it is the last commit on master. the difference is significant, because there are more than 900 commits between the `v-2-9-1-release` tag and `19fd5054b29af1b4e3b3278bfffbb6274c6c89f5` commit.

Reply:
I see every URL like https://github.com/conan-io/conan/archive/1.21.0.zip redirects to the codeload thing. I'd prefer to have the original one as it could be confusing.

Regarding the commit: Is there a way to know if that commit is part of the 2.9.1 release?

Reply:
It is not. The commit is dated 3 years after the release

Reply:
Maybe we can take a look at vcpkg to get some consensus about the versioning of this library

Reply:
VCPKG doesn't offer pthread-32, however, Conda does https://github.com/conda-forge/pthreads-win32-feedstock/blob/master/recipe/meta.yaml They are using ftp://sourceware.org

Reply:
@ericLemanissier could you check what is the commit corresponding to the sources Conda is using? I've seen that those are extracted from ftp://sourceware.org/pub/pthreads-win32/pthreads-w32-2-9-1-release.zip

Reply:
so, it appears that https://github.com/GerHobbelt/pthread-win32/ is an independant fork (no change since 2015), and that sources hosted on sourcreware are outdated (no change since 2013). The project is now hosted on https://sourceforge.net/projects/pthreads4w (still maintained by rosspjohnson). The version 3.0.0 was released in august 2018, so I will try to package this version instead.

Reply:
We have a recipe for that https://github.com/bincrafters/conan-pthreads4w

Reply:
@Croydon is right, we have different recipes.

Reply:
@ericLemanissier thanks for the investigation. It seems it would be better to package pthreads4w as @Croydon suggested